### ISSUE TITLE
UI Improvement: Start menu displays prefixes which slow reading down a lot and become chaotic

### ISSUE THREAD

**Issue by t4777sd** [QubesOS/qubes-issues#3910]
Please have a look at this screen shot for reference: https://www.qubes-os.org/attachment/wiki/QubesScreenshots/r2b2-kde-start-menu.png

You will see the start menu is organized in a hierarchy that has a bunch of things with the same prefixes. Example:

Domain: banking
Domain: personal
Domain: work

People are skimming this list left to right and having all the items with the same prefix slows down the reading. People are really looking for "banking", "personal" etc. Considering the frequency of times people would need to use the start menu, the confusion and time loss can be significant.

I propose instead to have an internal grouping in this menu:

---DOMAIN---
banking
personal
work

To clarify, by an internal grouping, I do not mean that the user hovers over "domain" and sub-menus popout. I mean that the positions stay exactly like they are now. However, "--DOMAIN--" header is placed above it and the relevant menu items "banking", "personal", etc, are placed under this. Perhaps all surrounded in a box with a different color /  borders so that it is obvious what items belong under the domain heading so there is no confusion for the menu items that belong after the domain section. 

In other words, instead of that menu being one long list of items using prefixes to differentiate different menu item classes it breaks up the menu into logical sections that are quicker to identify and utilize.

All the menu items that utilize the same prefixes can follow this model. For example, "Templates" would also be converted to this.

**The Benefits of This**

1) It will look much better. It won't look so cluttered. It gives some organizational structure to that main menu that is better than the prefix method being used now

2) It will be much more efficient to skim and quickly select the right items

3) It allows for longer names of domains, etc, because there is no longer a prefix that takes up space

**Comment by RefinedSoftwareLLC** [comment]
Mirrorway, nice idea. It takes up a little bit less height (maybe 0.5 rows per devider) and might be more asteticly pleasing. The issue with your idea is that it would be harder for new users & guests to understand, requiring a hover popup over each icon for them to understand what each icon means. And it may not be ovious to new users & guests that hovering over the icons explains them.

For either idea, having a popup when hovering over the icons or prefix dividers could display a popup that has more then just the prefix name, but also a short discription of its intended use, like disposibles do not save their changes, and that templates should not be used directly except for updates & configuration changes.

**Comment by mirrorway** [comment]
Or make the icons for template/system/dvm qubes have a unified theme.
I think all qubes icons should be variant of current appmenu icon (white Q on blue square).
but with "Q" replaced by "T" (for template), "A" (for appvm), etc...
instead of blue square, the square should be the assigned VM color.

This way the icon for each qube will be a cube (haha!), so you can tell immediately from the icon "this is a VM"...

**Comment by techgeeknz** [comment]
That's a great idea, mirrorway.  The same icon theme could then replace the two columns of icons in Qube Manager ("Type" and "Label") with one.  "dom0" can then be represented in both the appmenu and Qube Manager with the existing Q-in-box icon.

**Comment by ratpoison4** [comment]
Another improvement idea: display several columns of vms. If you have 100 VMs, it becomes increasingly long to scroll to VMs in the end of the list. If screen space was utilizes more wisely, even 500 VMs could be shown without need to scroll.

**Comment by UndeadDevel** [comment]
It seems this issue should get a `affects Q4.1` label and autoclosed on Q4.1 EOL, since Q4.2 replaces the old app menu with a new one that doesn't have this problem.
